composer.json

```diff
@@ -10,15 +10,13 @@
         }
     ],
     "require": {
-        "php": "^7.1|^8.0",
+        "php": "^8.0",
         "ext-json": "*",
-        "aws/aws-sdk-php": "^3.138",
         "keboola/common-exceptions": "^1.0",
-        "microsoft/azure-storage-blob": "^1.5",
-        "monolog/monolog": "^1.23|^2.0",
-        "symfony/filesystem": "^5.0|^6.0",
-        "symfony/http-foundation": "^5.0|^6.0",
-        "symfony/http-kernel": "^5.0|^6.0"
+        "monolog/monolog": "^2.0",
```

Comment:
AFAIK the reason we are locked on Monolog 2 is there was a breaking change in log record structure in Monolog 3 - introduction og `LogError` structure. Maybe this is the right time to switch that also?

Reply:
I guess now is as good a time as any :-) https://github.com/Seldaek/monolog/blob/main/CHANGELOG.md#300-rc1-2022-05-08